pkg/vsphere/extraconfig/encode.go

```diff
@@ -19,22 +19,36 @@ import (
 	"errors"
 	"fmt"
 	"net"
+	"os"
 	"reflect"
 	"sort"
 	"strconv"
 	"strings"
 	"sync"
 	"time"
 
-	log "github.com/Sirupsen/logrus"
+	"github.com/Sirupsen/logrus"
+
+	"github.com/vmware/vic/pkg/log"
 )
 
 var (
-	// EncodeLogLevel value
-	EncodeLogLevel = log.InfoLevel
 	ErrKeyNotFound = errors.New("key not found")
 )
 
+var encodeLogger = &logrus.Logger{
```

Comment:
Why the separate loggers for encode and decode? Generally I thought one logger per package was a good idea. Additionally you are setting log levels for both every time you set it for one, so you may be able to just use one logger for the extraconfig package.